src/platform/unittest/TlsTest.cpp

```diff
@@ -768,6 +771,7 @@ const CXPLAT_TLS_CALLBACKS TlsTest::TlsContext::TlsClientCallbacks = {
     TlsTest::TlsContext::OnPeerCertReceived
 };
 
+QUIC_CREDENTIAL_FLAGS TlsTest::SelfSignedCertParamsFlags = QUIC_CREDENTIAL_FLAG_NONE;
```

Comment:
Double-chck that `CxPlatGetSelfSignedCert` doesn't set flags on the object by default. I think it does, so you might not want to start with `non`

Reply:
I'm copying the default flags after we initialize the cert. This `none` is overwritten.